tile-json/map.js

```diff
@@ -0,0 +1,162 @@
+import { Loader } from '@googlemaps/js-api-loader';
+import axios from 'axios';
+import { GOOGLE_MAPS_TOKEN } from '@env';
+
+/**
+ * This example shows how to use the GeoJSOn response from our Vector Tile Server with Google Maps
```

Comment:
This example illustrates how the GeoJSON response from the tile service can be used in combination with Google Maps.